app/models/validations/date_validations.rb

```diff
@@ -59,6 +59,16 @@ def validate_startdate(record)
     if record["mrcdate"].present? && record.startdate < record["mrcdate"]
       record.errors.add :startdate, I18n.t("validations.setup.startdate.after_major_repair_date")
     end
+
+    location_inactive_status = inactive_status(record.startdate, record.location&.location_deactivation_periods, record.location&.available_from)
```

Comment:
Where does the location get chosen when creating a lettings log? I can't seem to actually pick one anywhere. I'm assuming it's possible, because in your comments at the beginning of this PR it seems like picking a tenancy start date (for a lettings log) triggers a validation error if either the scheme _or_ location is deactivated on the chosen date.

Reply:
You'd need to choose a scheme first and the next question would be location. There has to be more than 1 location available for that scheme tho for the location question to show up